File: CS/DxSample/Gallery/CustomGalleryControl.cs
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.Gallery;

namespace DxSample.Gallery {
    public class CustomGalleryControl :GalleryControl {
        protected override GalleryControlGallery CreateGallery() {
            return new CustomGallery(this);
        }
    }
}

File: VB/DxSample/Gallery/CustomGalleryControl.vb
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.Gallery

Namespace DxSample.Gallery

    Public Class CustomGalleryControl
        Inherits GalleryControl

        Protected Overrides Function CreateGallery() As GalleryControlGallery
            Return New CustomGallery(Me)
        End Function
    End Class
End Namespace
